common/models/user.js

```diff
@@ -119,11 +120,12 @@ module.exports = function(User) {
    * @returns {Object} The normalized credential object
    */
   User.normalizeCredentials = function(credentials, realmRequired, realmDelimiter) {
+    var caseSensitiveEmail = this.settings.caseSensitiveEmail;
```

Comment:
Yeah, but then I would have to add the same line under 137 as well (if a realm is required). Still worth it?